bot/src/wrapper/_communication.py

```diff
@@ -0,0 +1,170 @@
+from enum import IntEnum
+
+import aiohttp
+
+from ._api_schema import (  # Company
+    BatchCompaniesOutput,
+    CompanyGetIdOutput,
+    CompanyPatchIdInput,
+    CompanyPostInput,
+    RawShopItem,
+    ShopBuyInput,
+    ShopBuyOutput,
+    ShopGetOutput,  # Shop
+)
+from .error import AlreadyExistError, DoNotExistError, UnknownNetworkError, UserError
+
+
+class Status(IntEnum):
+    """Store the used http status code from the api."""
+
+    OK = 200
+    OK_CREATED = 201
+    BAD_REQUEST = 400
+    UNAUTHORIZED = 401
+    FORBIDDEN = 403
+    NOT_FOUND = 404
+    CONFLICT = 409
+
+
+class CompanyRawAPI:
+    """A bundle of API available to use for company endpoint."""
+
+    @staticmethod
+    async def create_company(address: str, token: str, src: CompanyPostInput) -> None:
+        """Create a company with the given details through a direct HTTP request."""
+        async with (
+            aiohttp.ClientSession(base_url=address, headers={"Authorization": token}) as session,
```

Comment:
Is there any way to re-use sessions so this doesn't have to be re-defined every time?

Reply:
I would provide a way to input a session and have a check of aiohttp internal to see if there are way to check if a session in open

(You should have a session from Pycord)

Reply:
I could check `self.closed`, it's possible

Reply:
Could make it so you make some sort of client that can then be stored on the bot side and re-used.

This isn't urgent though and better to focus on something new for now.

Reply:
Could do that as well